### PR TITLE
Publish DB image

### PR DESCRIPTION
**Pull request by dimakuz**
This PR does 2 things:

* Published DB image to the hub
* Allows downloading dump on DB initializtion if it is missing

### CODE REVIEW

File: db_docker/restore_db.sh [hunk 1]
```diff
@@ -5,6 +5,10 @@ set -x
 
 if [ "$RESTORE_DB" == "TRUE" ]; then
     echo "******DEV Env - PostgreSQL initialisation******"
+    if [ ! -f $DB_DUMP_PATH ]
+    then
+        python3 /download_dump.py
```
Comment (atalyaalon) [inline db_docker/restore_db.sh:10]: Did you check that downloading the dump is working this way? I had a hard time downloading the dump at restore time due to permissions to save the file if I remember correctly, that's why I made sure it happens in the Dockerfile instead.

File: db_docker/Dockerfile [hunk 2]
```diff
@@ -8,11 +8,13 @@ ARG RESTORE_DB='FALSE'
 ARG GDRIVE_URL='https://drive.google.com/uc?id='
 ARG DB_DUMP_PATH=/tmp/anyway_public.pgdump
 
-ENV GDRIVE_FILE_ID=$GDRIVE_FILE_ID
-ENV RESTORE_DB=$RESTORE_DB
-
 ADD ./download_dump.py /
 
-RUN if [ "$RESTORE_DB" == "TRUE" ] ; then python3 download_dump.py && chmod +rwx $DB_DUMP_PATH; else echo "RESTORE_DB!=TRUE, not downloading dump" ; fi
+RUN if [ -n "${GDRIVE_FILE_ID}" -a "${RESTORE_DB}" == "TRUE" ]; then python3 download_dump.py; else echo "No file ID provided or RESTORE_DB!=TRUE, not downloading dump" ; fi
```
Comment (atalyaalon) [inline db_docker/Dockerfile:13]: Did you check it's all working?
Also, is there any change we should do in the instructions of the [docs](docs/DOCKER.md)?

File: db_docker/restore_db.sh (hunk 1 above)
Comment (dimakuz) [reply db_docker/restore_db.sh:10]: I cheked downloading both at build and spin up time and I worked as expected.

**Comment by atalyaalon** [comment]
@dimakuz great work! @assafdayan can you take a look before merging?

**Comment by OriHoch** [comment]
@dimakuz I'm working on it to integrate all parts

**Comment by OriHoch** [comment]
commits are included in #1328

**Pull request closed without merging**